kin/account.py

```diff
@@ -316,42 +304,47 @@ def monitor_payments(self, callback_fn):
         """
         return self._client.monitor_account_payments(self.keypair.public_address, callback_fn)
 
-    # Internal methods
-
-    def _build_send_asset(self, asset, address, amount, memo_text=None):
-        """Build a tx to send asset to the account identified by the provided address.
+    def whitelist_transaction(self, payload):
+        """
+        Sign on a transaction to whitelist it
+        :param str payload: the json received from the client
+        :return: a signed transaction encoded as base64
+        :rtype str
+        """
 
-        :param str address: the account to send asset to.
+        # load the object from the json
+        if not isinstance(payload, dict):
```

Comment:
If the developer send me the data as a dict instead of a string, then I don't need to convert it using ```json.loads```. Just a way to avoid a common error people will have